tests/testthat/test-listings.R

```diff
@@ -137,50 +137,57 @@ testthat::test_that("regression test for keycols being lost due to `head()`", {
 testthat::test_that("column inclusion and ordering stuff", {
   ## key columns must be left-most k columns (#36)
   lsting <- as_listing(
-      df = ex_adae,
-      disp_cols = c("USUBJID", "AGE", "SEX", "RACE", "ARM", "AEDECOD", "AESEV", "AETOXGR"),
-      key_cols = c("ARM", "USUBJID", "AETOXGR")
+    df = ex_adae,
+    disp_cols = c("USUBJID", "AGE", "SEX", "RACE", "ARM", "AEDECOD", "AESEV", "AETOXGR"),
+    key_cols = c("ARM", "USUBJID", "AETOXGR")
   )
 
   mpfh <- matrix_form(head(lsting))
   exp <- mpfh$strings[1:4, 1:4]
   dimnames(exp) <- NULL
   testthat::expect_equal(exp,
-               matrix(c("Description of Planned Arm", "Unique Subject Identifier", "Analysis Toxicity Grade", "Age",
-                        "A: Drug X",                  "AB12345-BRA-1-id-134",      "2",                       "47",
-                        "",                           "",                          "",                        "47",
-                        "",                           "",                          "3",                       "47"),
-                      byrow = TRUE, nrow = 4))
+                         matrix(c("Description of Planned Arm",
+                                  "Unique Subject Identifier",
+                                  "Analysis Toxicity Grade", "Age",
+                                  "A: Drug X",
+                                  "AB12345-BRA-1-id-134",
+                                  "2",                       "47",
+                                  "",                           "",
+                                  "",                        "47",
+                                  "",                           "",
+                                  "3",                       "47"),
+                                byrow = TRUE, nrow = 4))
   ## key cols don't need to be repeated in disp_cols
```

Comment:
I did this bc also lintr was complaining about this one. We should consider using testthat e3 with snapshots